src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -103,7 +107,6 @@ interface ImageFormProps {
 		slug: string,
 		version: string | null,
 	) => Promise<string | undefined>;
-	modalActions?: ModalAction[];
```

Comment:
I think it's worth making this a separate commit with `minor` change type.

Reply:
no problem